docs/model-monitoring/index.md

```diff
@@ -0,0 +1,72 @@
+to the same target. You can use `get_offline_features` to read the data from that feature set. 
+
+Read also how {ref}`model monitoring supports the gen AI model server<genai-mmonitor>`.
+
+## Streaming platforms and credentials
+
+Model monitoring supports open-source streaming platforms such as Kafka, TDEngine, MySQL (8.0 and higher), in addition to integration with the Iguazio AI platform V3IO data layer. 
+Before you deploy the model monitoring or serving function, you need to {py:meth}`set the credentials <mlrun.projects.MlrunProject.set_model_monitoring_credentials>`. 
+
+## Model monitoring applications
+
+When you call `enable_model_monitoring` on a project, by default MLRun deploys te onitoring app, `HistogramDataDriftApplication`, which is 
+tailored for classical ML models (not LLMs, gen AI, deep-learning models, etc.). It includes:
+* Total Variation Distance (TVD) &mdash; The statistical difference between the actual predictions and the model's trained predictions.
+* Hellinger Distance &mdash; A type of f-divergence that quantifies the similarity between the actual predictions, and the model's trained predictions.
+* Kullback–Leibler Divergence (KLD) &mdash; The measure of how the probability distribution of actual predictions is different from the second model's trained reference probability distribution.
+
+You can create your own model monitoring applications, for LLMs, gen AI, deep-learning models, etc., based on the class {py:meth}`mlrun.model_monitoring.applications.ModelMonitoringApplicationBaseV2`. 
+
+Projects are used to group functions that use the same model monitoring application. You first need to create a project for aspecific application. 
+Then you disable the default app, enable your customer app, and create and run the functions. 
+
+See the:
+- User flow example **in ??????**.
+- Custom apps in **?????**.
+
+The basic flow for classic ML and other models is the same, but the apps and the infer requests are different.
+
+
+
+
+## Model and model monitoring endpoints 
+
+Each unique combination of runtime function (a deployed Nuclio function) and model has an endpoint and a corresponding {py:meth}`model endpoint <mlrun.model_monitoring.api.get_or_create_model_endpoint>`. 
+All model monitoring endpoints are presented in the UI with information about the actual inference, including data on the inputs, outputs, and results.
+The Model Endpoints tab presents the overall metrics. From there you can select an endpoint and view the Overview, Features Analysis, and the Metrics tabs. 
+Metrics are grouped under their applications. After you select the metrics and the timeframe, you get a histogram showing the number of occurrences/values range, and a timeline 
+graph of the metric and the threshold. Any alerts are shown in the upper-right corner of the metrics box. For example:
+
```

Comment:
i think we should explain here the connection between the model monitoring functions to the model endpoints, that each model that server in a model serving function we creating a model endpoint and the model endpoint is related to a feature set that manage the model endpoint stats @davesh0812 let me know what do you think.
it can wrriten in diffrent place but then remove this line `You can use get_offline_features to read the data from that feature set`